shared/message/platform/webots/messages.proto

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+syntax = "proto3";
+
+package message.platform.webots;
+
+// Copy of cyberbotics/webots projects/samples/contests/robocup/controllers/player/messages.proto
```

Comment:
Not sure if the comments I have left for this file can be addressed easily if we don't have more info than just the copy/paste job. 
Also, this filepath doesn't exist any more, unless I am confused and you didn't mean that this was on the `cyberbotics/webots` repo on GitHub. Not sure what they have done with this file at all.

Reply:
On [the webots' controller](https://github.com/cyberbotics/webots/pull/2888)